Makefile

```diff
@@ -0,0 +1,50 @@
+PG_CONFIG   ?= $(shell which pg_config)
+DISTNAME     = $(shell grep -m 1 '^name' pg_search/Cargo.toml | sed -e 's/[^"]*"\([^"]*\)",\{0,1\}/\1/')
+DISTVERSION  = $(shell cargo tree --prefix=none --edges=normal | grep 'pgrx v' | sed 's/pgrx v//')
+PGRXV        = $(shell perl -nE '/^pgrx\s+=\s"=?([^"]+)/ && do { say $$1; exit }' Cargo.toml)
+PGV          = $(shell perl -E 'shift =~ /(\d+)/ && say $$1' "$(shell $(PG_CONFIG) --version)")
+
+all: package
+
+# Print the current Postgres version reported by pg_config.
+.PHONY: pg-version
+pg-version:
+	@echo $(PGV)
+
+# Print the current PGRX version from Cargo.toml
+.PHONY: pg-version
+pgrx-version:
+	@echo $(PGRXV)
+
+# Install the version of PGRX specified in Cargo.toml.
+.PHONY: install-pgrx
+install-pgrx: pg_search/Cargo.toml
+	@cargo install --package pg_search --locked cargo-pgrx --version "$(PGRXV)"
+
+# Initialize pgrx for the PostgreSQL version identified by pg_config.
+.PHONY: pgrx-init
+pgrx-init: pg_search/Cargo.toml
+	@cargo pgrx init "--pg$(PGV)"="$(PG_CONFIG)"
+
+# Install pg_search into the PostgreSQL cluster identified by pg_config.
+.PHONY: install
+install:
+	@cargo pgrx install --package pg_search --release --pg-config "$(PG_CONFIG)"
+
+# Build pg_search for the PostgreSQL cluster identified by pg_config.
+.DEFAULT_GOAL: package
+package:
+	@cargo pgrx package --package pg_search --pg-config "$(PG_CONFIG)"
+
+META.json: META.json.in pg_search/Cargo.toml
+	@sed "s/@CRATE_VERSION@/$(DISTVERSION)/g" $< > $@
+
+$(DISTNAME)-$(DISTVERSION).zip: META.json
+	git archive --format zip --prefix $(DISTNAME)-$(DISTVERSION)/ --add-file $< -o $(DISTNAME)-$(DISTVERSION).zip HEAD
+
+# Create a PGXN-compatible zip file.
+dist: $(DISTNAME)-$(DISTVERSION).zip
+
+# Delete files generated while making a PGXN-compatible zip file.
+clean:
+	@rm -rf META.json $(DISTNAME)-$(DISTVERSION).zip
```

Comment:
For a proper `clean` command you should add `rm -rf targets`, I think.

Reply:
Decided to go with `PGXS`and set the cleanup variable.